bluemira/equilibria/coils/_grouping.py

```diff
@@ -283,6 +284,26 @@ def from_group_vecs(cls, eqdsk: EQDSKInterface):
         pfcoils = []
         cscoils = []
         passivecoils = []
+        if eqdsk.ncoil < 1:
+            grid = Grid.from_eqdsk(eqdsk)
+            dum_xc = [np.min(grid.x), np.max(grid.x), np.max(grid.x), np.min(grid.x)]
+            dum_zc = [np.min(grid.z), np.min(grid.z), np.max(grid.z), np.max(grid.z)]
+            for i in range(4):
```

Comment:
What's the rationale for 4 dummy coils here?

Reply:
A coil in each corner of the grid seemed ok as they should not be used. But I could change this.